src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs

```diff
@@ -81,6 +81,8 @@ public async Task<IActionResult> ViewStandard(int larsCode)
             
             model.EditContactDetailsUrl = Url.RouteUrl(RouteNames.GetCourseContactDetails, new { Ukprn, larsCode });
 
+            model.ConfirmRegulatedStandardUrl = Url.RouteUrl(RouteNames.ConfirmRegulatedStandard, new { Ukprn, larsCode }, Request.Scheme, Request.Host.Value);
```

Comment:
What is the purpose of Request.Scheme and Request.Host? Other url seems to be working with out adding these information.